galsim/wcs.py

```diff
@@ -323,7 +323,19 @@ def toImage(self, arg, **kwargs):
 
                >>> image_pos = wcs.toImage(world_pos)
 
-        2. The second converts a surface brightness profile (a GSObject) from world
+           Equivalent to wcs.posToImage(world_pos).
+
+        2. The second is nearly the same, but takes either u and v values or ra and dec values
+           (depending on the kind of wcs being used) directly and returns x and y values.
+           For this version, the inputs maybe be numpy arrays, in which case the returned values
+           are also numpy arrays.
+
+               >>> x, y = wcs.toImage(u, v)                 # For EuclideanWCS types
+               >>> x, y = wcs.toImage(ra, dec, units=units) # For CelestialWCS types
```

Comment:
It would be nice to make these explicit keywords to avoid the order of arguments confusion.  How can the user know whether it should be `u, v` or `v, u`?
```
x, y = wcs.toImage(u=u, v=v)
x, y = wcs.toImage(ra=ra, dec=dec, units=units)
```
Of course the return values also have this ambiguity `x, y` or `y, x`?

Reply:
> How can the user know whether it should be u, v or v, u?

Read the documentation?  :) 
I think I'm pretty reticent to add a big infrastructure to parse both args and kwargs for this to make sure all possible ways of specifying things work correctly.  I did point to which more explicitly named functions (with explicit named arguments) are equivalent.  Do you think this is sufficient for those who want their code to be more self-documenting than the overloaded `toImage` and `toWorld`?

Reply:
I don't feel strongly about it, go with what you have.